crates/re_viewer/src/ui/space_view.rs

```diff
@@ -62,12 +62,7 @@ impl SpaceView {
         space_path: &EntityPath,
         queries_entities: &[EntityPath],
     ) -> Self {
-        let display_name = if queries_entities.len() == 1 {
-            // A single entity in this space-view - name the space after it.
-            queries_entities[0]
-                .last()
-                .map_or_else(|| "/".to_owned(), |part| part.to_string())
-        } else if let Some(name) = space_path.iter().last() {
+        let display_name = if let Some(name) = space_path.iter().last() {
```

Comment:
With this PR we now sometimes get more things named after root (`/`):

![image](https://user-images.githubusercontent.com/1148717/226649689-cc39fc21-f793-4b5a-b6df-d774be880f9c.png)

…but I still think that this PR is an improvement.

But it would be good to add a comment here for why this is the better choice, so we don't regress here.